pkg/controller/installation/core_controller.go

```diff
@@ -879,14 +884,43 @@ func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.
 	}
 
 	// Everything is available - update the CRD status.
+	mtu, err := readMTUFile()
+	if err != nil {
+		r.SetDegraded("error reading network MTU", err, reqLogger)
+		return reconcile.Result{}, err
+	}
+	if mtu == 0 {
```

Comment:
We could change this so that instead of blocking the controller, we just don't write the status field.

That would remove dependence on MTU for other controllers to start, and also dependence on the `/var/lib/calico/mtu` file being mounted properly.

Reply:
Yeah, I guess it comes down to whether or not OpenShift will handle the MTU status field changing gracefully or not (I hope it does).